config/ppo/Hallway.yaml

```diff
@@ -5,7 +5,7 @@ behaviors:
       batch_size: 128
       buffer_size: 1024
       learning_rate: 0.0003
-      beta: 0.01
+      beta: 0.03
```

Comment:
Should we add in the documentation that it may be good practice to increase beta with LSTM? Or do we think that this is a peculiarity of Hallway?

Reply:
I think it's b/c of Hallway's sparse reward and reward structure that has a bunch of local maxima (at least 2, given the reward curve). Hard to say if we'd need to increase beta for other envs.
Our betas might just be too low across-the-board.